### DISCUSSION TITLE
FAO Roberthh - Re you modded SHT30 library module.

### DISCUSSION THREAD

**Discussion by Ajb2k3** [micropython/micropython#10660] [General]
First off, thank you for fixing the code and showing the forum user how to use it.
I'm writing a micropython basics book and using your library as part of the I2C reference section. I don't really want to post the whole code in the book and was wondering if you had a GitHub page for it I could link to instead please.

**Comment by robert-hh** [comment]
The initial source is here: https://github.com/rsc1975/micropython-sht30/blob/master/sht30.py. A little bit older than my version, and with quite a few obsolete `;` at the line ends and a version of I2C requiring start() and stop().

**Reply by Ajb2k3** [reply]
yeh, I was having a lot of trouble getting it to work but I was hoping for a link to your version.

**Reply by robert-hh** [reply]
It's raining outside at 5°C, not very comfy. So I'll make a copy at github.

**Reply by Ajb2k3** [reply]
Thank you, it just makes it nicer for references and programmer.
Damn horrid weather!

**Reply by robert-hh** [reply]
It's up at https://github.com/robert-hh/SHT30

**Reply by Ajb2k3** [reply]
Thanks Mate, corrected the section of my book to your page!
Thank you very much for your help.

**Reply by robert-hh** [reply]
I assume you have referred to Awesome Micropython at https://awesome-micropython.com/ as well. There is a link to simple SHT31 driver, which may work as well for the SHT30.
